src/services/chat/prepopulated.js

```diff
@@ -41,17 +41,17 @@ module.exports = async (id, users) => {
             better_channel_member: [
               {
                 user: {
-                  ...ownUser
+                  ...ownUser.defaulValues
                 },
-                user_id: ownUser?.id,
+                user_id: ownUser?.user_id,
                 is_anonymous: false,
                 updated_at: new Date().toISOString()
               },
               {
                 user: {
-                  ...user
+                  ...user.defaulValues
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The same typo is present here as well. It should be `defaultValues` if it exists on the `user` object.
```diff
- ...user.defaulValues
+ ...user.defaultValues
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->